metricbeat/docs/fields.asciidoc

```diff
@@ -20707,7 +20707,7 @@ Stats scraped from a Prometheus endpoint.
 
 
 
-*`prometheus.labels`*::
+*`prometheus.labels.*`*::
```

Comment:
Glad you found the convention ;-)

Reply:
why is this a convention? I think this is the only `type: object` with `object_type: keyword` that uses `*`, example: https://github.com/elastic/beats/blob/master/libbeat/_meta/fields.ecs.yml#L23

Reply:
We investigated this offline, it seems that both options generate equivalent dynamic templates (`.*` is appended if there is no `*`). But the new fields validator for the results of testdata requires the `*`.

Reply:
Jaime is correct, it's for the new test framework.